sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2675,6 +2675,15 @@ object SQLConf {
       .checkValues(LegacyBehaviorPolicy.values.map(_.toString))
       .createWithDefault(LegacyBehaviorPolicy.EXCEPTION.toString)
 
+  val LEGACY_ALLOW_MODIFY_ACTIVE_SESSION =
```

Comment:
Should this be a static config? I am not quite sure if there is any case that one SparkSession can set this to true, then other SparkSessions are affected silently.

Reply:
sounds reasonable, will update.

Reply:
done